core/comm.go

```diff
@@ -26,13 +28,13 @@ func (c *Client) TxnBegin() {
 	c.isTxn = true
 }
 
-func (c *Client) TxnExec() []byte {
+func (c *Client) TxnExec(dh *handlers.DiceKVstoreHandler) []byte {
```

Comment:
Instead of calling it handler, let's name our storage engine and use that here.

I propose the name - `ozone`